test/plugin_test.go

```diff
@@ -37,6 +40,39 @@ func (s *CLITestSuite) TestPlugin() {
 	}
 }
 
+func (s *CLITestSuite) TestPluginUninstall() {
+	req := require.New(s.T())
+	filename := "test/fixtures/input/plugin/confluent-test-plugin-uninstall.sh"
+	_, err := os.OpenFile(filename, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0777)
```

Comment:
I think there are built-in functions in the Go standard library that set these automatically.